WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -666,6 +654,27 @@ private extension OrderDetailsViewController {
 
         return nil
     }
+
+    func deleteTracking(_ tracking: ShipmentTracking) {
+        let siteID = viewModel.order.siteID
+        let orderID = viewModel.order.orderID
+        let trackingID = tracking.trackingID
+
+        let deleteTrackingAction = ShipmentAction.deleteTracking(siteID: siteID,
+                                                                 orderID: orderID,
+                                                                 trackingID: trackingID) { [weak self] error in
+                                                                    if let error = error {
+                                                                        DDLogError("⛔️ Delete Tracking Failure: orderID \(orderID). Error: \(error)")
```

Comment:
When creating error messages, it also helps to say where the error is from. So I would say "⛔️ Order Details - Delete Tracking Failure: orderID \(orderID). Error: \(error)" to make it easier when troubleshooting issues from a emailed Error log. If you find hints like that useful, think about adding it here too.